tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -888,6 +888,16 @@ class WskBasicUsageTests extends TestHelpers with WskTestHelpers {
       "Cannot save action sequences")
   }
 
+  it should "reject an invoke with the wrong parameters set" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
Actually this is a duplicate test and can be deleted. The same test occurs here https://github.com/apache/incubator-openwhisk/blob/master/tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala#L1554.

Reply:
done